CONTRIBUTING.md

```diff
@@ -37,9 +37,10 @@ Use Gitpod for a cloud-based development environment:
 4. Create a new branch: `git checkout -b my-feature-branch`
 5. Make your changes and commit: `git add .` and `git commit -m "Description of changes"`
 6. Test with Molecule: `make molecule-test` or `make molecule-converge`
-7. Push your changes: `git push origin my-feature-branch`
-8. Create a pull request on GitHub
-9. Wait for a review
+7. Test with linters(otpional): `make lint`
```

Comment:
```suggestion
7. Test with linters: `make lint`
```

Reply:
I would recommend to run the linters first and only then the tests.

Reply:
I completely agree that linters should be integrated into tests and made mandatory, meaning that code cannot be validated and pushed unless it passes the linter check. However, since we are not yet fully clean on the linters, I think it's better to leave them optional for now and personally check the pull requests.

My belief is that it's the responsibility of the projects (and therefore the project managers) to ensure that the necessary tools are in place for contributors. Therefore, I suggest that we leave the linters optional for the time being and make them mandatory directly in the Makefile once the work on the linters is done.

What do you think of this approach?